### PR TITLE
Add example simple mirror

### PR DESCRIPTION
**Pull request by jobdewitte**
added a example for a simple (radially symmetric) mirror configuration.

### CODE REVIEW

File: examples/simple-mirror.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+boundary = G.Path.line([0, 0, -1], [2, 0, -1]).line_to([2, 0, 1]).line_to([0.3, 0., 1])
+mirror = G.Path.line([0., 0., 0.], [1., 0., 0.])
+
+boundary.name = 'boundary'
+mirror.name = 'mirror'
+
+mesh = (boundary + mirror).mesh(mesh_size_factor=45)
+
+# Show the generated mesh, with the given electrode colors.
+P.plot_mesh(mesh, ground='green', lens='blue', show_normals=True)
```
Comment (leon-vv) [inline examples/simple-mirror.py:21]: Please change names to correspond with 'boundary', 'mirror'

Comment (leon-vv) [reply examples/simple-mirror.py:21]: There is also no point in showing normals if you are using only voltage excitations

Comment (jobdewitte) [reply examples/simple-mirror.py]: fixed. see new commit

**Comment by leon-vv** [comment]
Let's discuss this example in our meeting at 17h, I feel like a 'realistic' mirror (e.g. without a flat electrode in the middle) would serve as a more realistic example

File: examples/simple-mirror.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+excitation = E.Excitation(mesh, E.Symmetry.RADIAL)
+
+excitation.add_voltage(mirror=-110, boundary=0.0)
+
+field = S.solve_direct(excitation)
+field_axial = FieldRadialAxial(field, 0.02, 4, 600)
+
+#NOTE: discrepancy between interpolation and integration near z=0
```
[the remark this answers is not in the document]
Comment (leon-vv) [reply examples/simple-mirror.py:30]: Kind of. You compute the radial expansion between 0.02 and 4, and subsequently compute the potential exactly at 0.02, apparently the code is currently such that the range is non inclusive (0.02, 4) such that for 0.02 it returns the default value 0.00. We could try to fix this

[the remark this answers is not in the document]
Comment (leon-vv) [reply examples/simple-mirror.py:30]: Fixed by #103

**Pull request closed without merging**